gluten-core/src/main/scala/io/glutenproject/backendsapi/SparkPlanExecApi.scala

```diff
@@ -231,6 +231,14 @@ trait SparkPlanExecApi {
       attributeSeq: Seq[Attribute]): ExpressionTransformer =
     new NamedStructTransformerBase(substraitExprName, original, attributeSeq)
 
+  def genEqualNullSafeTransformer(
+      substraitExprName: String,
+      left: ExpressionTransformer,
+      right: ExpressionTransformer,
+      original: EqualNullSafe): ExpressionTransformer = {
+    new BinaryExpressionTransformer(substraitExprName, left, right, original)
```

Comment:
Maybe this is not needed becausae Velox backend and CH can share the same transformer.

Reply:
Thanks for advise! Done.